tests/integration/components/bs-modal-test.js

```diff
@@ -312,5 +312,31 @@ module('Integration | Component | bs-modal', function (hooks) {
       await settled();
       assert.dom('.modal').doesNotExist();
     });
+
+    test('it passes along fullscreen class attribute', async function (assert) {
+      this.set('fullscreen', 'full');
```

Comment:
If I understand API documentation correctly, `full` is not a supported option. It should be `true` for this test case, isn't it?

```suggestion
      this.set('fullscreen', true);
```

Reply:
Fixed